Makefile

```diff
@@ -90,9 +92,24 @@ push_translations:
 	$$(npm bin)/edx_reactifex $(transifex_temp) --comments --v3-scripts-path
 	./node_modules/@edx/reactifex/bash_scripts/put_comments_v3.sh
 
-pull_translations: ## must be exactly this name for edx tooling support, see ecommerce-scripts/transifex/pull.py
-	# explicit list of languages defined here and in currentlySupportedLangs.jsx
-	tx pull -t -f --mode reviewed --languages="ar,fr,es_419,zh_CN"
+# Pulls translations from Transifex.
+pull_translations:
+ifeq ($(OPENEDX_ATLAS_PULL),)
+else
+endif
```

Comment:
is this intentional? it seems like lines 101-112 are handling this correctly

Reply:
this is a typo. I'll remove it.